salt/modules/junos.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
 '''
-Module for interfacing to Junos devices.
+Module to interact with Junos devices.
 '''
 
 # Import python libraries
-from __future__ import absolute_import
 import logging
 import json
+import os
+import salt
```

Comment:
Same comment here as in the other module. If we can limit this to specifically the module(s) used, it would be better.